lib/jackal/callback.rb

```diff
@@ -108,7 +108,8 @@ def forward(payload, dest=nil)
       source = Carnivore::Supervisor.supervisor[dest]
       if(source)
         info "Forwarding payload to output destination... (#{source})"
-        debug "Forwarded payload: #{payload.inspect}"
+        output = Carnivore::Config.get(:pretty_print_payload) ? payload.pretty_inspect : payload.inspect
```

Comment:
gotcha; yeah, this is what I originally had in mind.  glad to get confirmation.  I'll update